lib/components/overlays/quick_settings/quick_settings_overlay.dart

```diff
@@ -117,6 +124,46 @@ class _QuickSettingsOverlayState extends State<QuickSettingsOverlay>
   }
 }
 
+PageRouteBuilder customRouteTransition(Widget screen) {
```

Comment:
make this private possibly and/or consider moving it inside the state class for the overlay